reproductions/schema/json-schema/index.js

```diff
@@ -0,0 +1,8 @@
+import { JSONSchemaBridge } from 'uniforms-bridge-json-schema';
+
+import { createValidator } from './validator';
+import schema from './schema';
+
+const schemaValidator = createValidator(schema);
```

Comment:
I'm just wondering if it does make sense to put JSON schema in three files. If the example schema will be formatted differently it'll take a few lines less.